### PR TITLE
Portal 1003/uswds spacing

### PR DESCRIPTION
**Pull request by cfarmer-fearless**
Import unit spacing token function from USWDS Sass modules.  Also refactors spacing CSS properties such as font size, width, height, padding, and margin to use the `units` token function.  See USWDS docs for more details: https://designsystem.digital.gov/design-tokens/spacing-units/

### CODE REVIEW

File: packages/cdc-react/src/scss/_palettes.scss [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
-  "cyan": $primary-cyan,
-  "green": $primary-green,
-  "indigo": $primary-indigo,
-  "orange": $primary-orange,
-  "pink": $primary-pink,
-  "purple": $primary-purple,
-  "slate": $primary-slate,
-  "teal": $primary-teal,
+  "amber": colors.$primary-amber,
+  "blue": colors.$primary-blue,
```
Comment (sublettk) [inline packages/cdc-react/src/scss/_palettes.scss:5]: Love this of prefixing with "colors". It feels much more intuitive.

File: packages/cdc-react/vite.config.ts [hunk 2]
```diff
@@ -15,6 +15,13 @@ export default defineConfig({
     environment: "jsdom",
     setupFiles: "./src/tests/setup.ts",
   },
+  css: {
+    preprocessorOptions: {
+      scss: {
+        includePaths: ["./node_modules/@uswds/uswds/packages"],
```
Comment (sublettk) [inline packages/cdc-react/vite.config.ts:21]: I'm concerned that referencing the alias like this directly from `node_modules` may cause some side effects. It seems a bit unconventional.

File: packages/storybook/vite.config.js [hunk 3]
```diff
@@ -9,7 +9,8 @@ export default defineConfig({
   resolve: {
     alias: {
       '@us-gov-cdc/cdc-react': path.resolve(__dirname, '../cdc-react/src/components/index.ts'),
-      '@us-gov-cdc/cdc-react-icons': path.resolve(__dirname, '../cdc-react-icons/src/components/index.ts')
+      '@us-gov-cdc/cdc-react-icons': path.resolve(__dirname, '../cdc-react-icons/src/components/index.ts'),
+      'uswds-core': path.resolve(__dirname, '../cdc-react/node_modules/@uswds/uswds/packages/uswds-core')
```
Comment (sublettk) [inline packages/storybook/vite.config.js:13]: I'm concerned that referencing the alias like this directly from `node_modules` may cause some side effects. It seems a bit unconventional.

File: packages/cdc-react/vite.config.ts (hunk 2 above)
Comment (cfarmer-fearless) [reply packages/cdc-react/vite.config.ts:21]: I also hate it.  Open to suggestions.

File: packages/storybook/vite.config.js (hunk 3 above)
Comment (cfarmer-fearless) [reply packages/storybook/vite.config.js:13]: I also hate it.  Open to suggestions.

Comment (ian-sawyer) [reply packages/storybook/vite.config.js:13]: It's been fairly normal for a while in bundlers. I think it is ok for now.

File: packages/cdc-react/vite.config.ts (hunk 2 above)
Comment (sublettk) [reply packages/cdc-react/vite.config.ts:21]: Let's keep it this way. Bootstrap also has this setup. [Bootstrap & Vite](https://getbootstrap.com/docs/5.2/getting-started/vite/)

**Review by ian-sawyer — APPROVED** [review]

**Pull request merged**